.gitignore

```diff
@@ -29,3 +29,9 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+"sendgrid.env" 
+"sendgrid.env" 
+"sendgrid.env" 
+
+# Ignore application configuration
+/config/application.yml
```

Comment:
@rishabhguptarishi make a duplicate file applicatoin.yml.example which will be in the repositories. That file will have keys, and example values